setup.py

```diff
@@ -1,35 +1,32 @@
-"""
-Setup.py for superset-patchup
-"""
 from setuptools import setup, find_packages
 
 setup(
     name='hq_superset',
-    version='0.2.0',
-    description='CommCareHQ Superset Integration',
+    version='0.3.0',
+    description='CommCare HQ Superset Integration',
     license='Apache2',
-    author='Dimagi Inc',
+    author='Dimagi Inc.',
     author_email='sreddy@dimagi.com',
     url='https://github.com/dimagi/hq_superset',
     packages=find_packages(exclude=['docs', 'tests']),
     include_package_data=True,
     install_requires=[
-        # Werkzeug 2.1 doesn't work, so pin it
-        'Werkzeug==2.0.0',
-        'jinja2==3.0.3',
-        'dimagi-superset==2.0.1',
-        'authlib==1.0.1',
-        'requests==2.28.1',
-        'psycopg2==2.9.3',
-        'WTForms==2.3.3',
-        'cryptography==37.0.4',
-        'sentry-sdk==1.9.10',
+        'apache-superset==3.1.0',
+        # Dependencies based on Superset 3.1.0 where applicable
+        'Authlib==1.3.0',
         'celery==5.2.7',
+        'cryptography==41.0.2',
+        'jinja2==3.1.2',
+        'psycopg2==2.9.6',
+        'requests==2.31.0',
+        'sentry-sdk==1.39.2',
```

Comment:
If we use `sentry-sdk[flask]`, then we don't have to install `blinker`

Reply:
1f4763b